### DISCUSSION TITLE
development practices: branches

### DISCUSSION THREAD

**Discussion by chausch** [ppooll-dev/ppooll#66] [General]
i just realized i've been using branches the wrong way. @the-steech you did it right from the start. @klausfilip the way is to open an issue and create the branch from the right sidebar from within the issue. you then work on this specific issue in this branch and close and delete the branch when it's done.
is that about right @the-steech ?
i was understanding the whole branch concept much more in a sense of "manual" control. but it makes sense this way and is way faster …

**Comment by the-steech** [comment]
Really hard to say "wrong" or "right" way, but I think this helps keeps code separate and issues organized :)

**Comment by the-steech** [comment]
Some issues where you make the same change to many acts could also be one branch, so it all falls under one category.  Maybe we could have a zoom call check in at some point to just review how all the new git stuff is working out !

**Comment by chausch** [comment]
yeah let's zoom soon!
